Source/Core/Matrix2.js

```diff
@@ -57,13 +59,8 @@ define([
      */
     Matrix2.pack = function(value, array, startingIndex) {
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(value)) {
-            throw new DeveloperError('value is required');
-        }
-
-        if (!defined(array)) {
-            throw new DeveloperError('array is required');
-        }
+        Check.typeOf.object(value, 'value');
+        Check.defined(array, 'array');
```

Comment:
The `Check.defined` was used in [`Cartesian3`](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/Cartesian3.js#L150).

All other changes resolved.